packages/vscode-extension/package.nls.json

```diff
@@ -4,5 +4,6 @@
     "spyglassmc.commands.showCacheRoot.title": "Open Cache Folder",
     "spyglassmc.description": "Heavy language features for Minecraft: Java Edition data pack files",
     "spyglassmc.displayName": "Datapack Helper Plus by Spyglass",
+    "spyglassmc.env.gameVersion": "The Minecraft version to use. When set to auto, the version is inferred from the `pack.mcmeta`. Other versions can be specified in a `spyglass.json` config file.",
```

Comment:
what does `nls` stand for in this file's name?

Reply:
"National Language Support" or "Native Language Support", VS Code's choice of word for i18n. This file is picked up by VS Code automatically for localization.